documentation/source/development/design/labeling.rst

```diff
@@ -0,0 +1,166 @@
+    plat, platform, platform specific issue
+
+Some labels are mutual to each other like ``org:in progress`` and ``org:on hold``. Also some labels should not be specified on closed issues like ``org:in progress``.
+
+.. note::
+
+    Github will use black as font color if the background is too light.
+
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| Preview             | Category | Name          | Description                                                                        | Color   |
++=====================+==========+===============+====================================================================================+=========+
+| |cat:benchmark|     | cat      | benchmark     | benchmarking                                                                       | #C0E169 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |cat:bug|           | cat      | bug           | bug/error/mistake which limits the program                                         | #FF0000 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |cat:crash|         | cat      | crash         | something causes the program to crash                                              | #FF0000 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |cat:dependency|    | cat      | dependency    | dependency management                                                              | #BFD4F2 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |cat:dev tools|     | cat      | dev tools     | building/ compiling the program, cmake configuration and general development tools | #78A600 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |cat:documentation| | cat      | documentation | documentation                                                                      | #D1D100 |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |cat:enhancement|   | cat      | enhancement   | enhancement/requested feature/update of existing features                          | #0025FF |
++---------------------+----------+---------------+------------------------------------------------------------------------------------+---------+
+| |cat:performance|   | cat      | performance   | performance                                                                        | #FF6868 |
```

Comment:
Maybe right now a lighter color might be better, as it is not of high priority? Later when the priority increases, we could change it to a darker color...